releases/development/master/extra/install-chef-suse.sh

```diff
@@ -714,15 +714,28 @@ fi
 
 echo_summary "Starting required services"
 
+if [ $(rpm -q --qf "%{version}" rubygem-chef | cut -d . -f 1) -ne 10 ]; then
+    die "Please remove the frame_max setting in rabbitmq!"
+fi
+
 # Write default RabbitMQ configuration (if the package doesn't provide one).
 if [ ! -f /etc/rabbitmq/rabbitmq.config ] ; then
     cat << EOF > /etc/rabbitmq/rabbitmq.config
 [
  {rabbit,
-  [{disk_free_limit, 50000000}]
+  [
+    {disk_free_limit, 50000000},
+    {frame_max, 0}
+  ]
  }
 ].
 EOF
+else
+    # allow unlimited frame size, as the version of bunny required by chef
+    # doesn't know how to deal with frames; and this setting is enforced with
+    # rabbitmq >= 3.0.0 (see
+    # http://www.rabbitmq.com/blog/2012/11/19/breaking-things-with-rabbitmq-3-0/)
+    sed -i "s/%% {frame_max, 131072},/{frame_max, 0},/g" /etc/rabbitmq/rabbitmq.config
```

Comment:
Hrm, it doesn't just work (need to remove the trailing `,`). Would probably be more clever to add this at the end of the rabbit section in the config. Will need some sed magic for that, I guess :-)